src/renderer/hooks/useDialog.ts

```diff
@@ -0,0 +1,13 @@
+import { useContext } from 'react'
+
+import { DialogContext } from '../contexts/DialogContext'
+
+export function useDialog() {
```

Comment:
```suggestion
export default function useDialog() {
```

This makes more sense I think?

Reply:
would need to be changed in files that import it too I guess?

Reply:
yeah